### DISCUSSION TITLE
Adult Swim Live Channels

### DISCUSSION THREAD

**Discussion by TheKingMateo** [iptv-org/community#1114] [Q&A]
What happened to the turnermedia links, I'm no longer able to watch adult swim east or west live through apps like tivimate since it requires a playlist?

**Comment by BellezaEmporium** [comment]
Simple : DMCA request from WBD.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/16839

**Reply by TheKingMateo** [reply]
Saw, thx still!
